v3/integration/config.json

```
@@ -426,6 +426,8 @@
       "ErrCount": 2
     },
     "e_ev_country_name_missing": {},
+    "e_ev_invalid_business_category": {
+      "ErrCount": 11306
```

```suggestion
      "ErrCount": 11306
      }
```

This time I forgot a character ;)